content/en/sensitive_data_scanner/_index.md

```diff
@@ -0,0 +1,178 @@
+1. Click **Create**.
+
+### Add scanning rules
+
+A scanning rule determines what sensitive information to match within the data defined by a scanning group. You can add predefined scanning rules from Datadog's Scanning Rule Library or create your own rules using regex patterns. The data is scanned at the point of ingestion. For logs, the scan is done before indexing.
+
+For Terraform, see the [Datadog Sensitive Data Scanner rule][6] resource.
+
+To add scanning rules:
+
+1. Navigate to [Sensitive Data Scanner][5].
+1. Click the scanning group where you want to add the scanning rules.
+1. Click **Add Scanning Rule**. Alternatively, click the **Add** dropdown menu on the top right corner of the page and select **Add Scanning Rule**.
+1. Select whether you want to add a library rule or create a custom scanning rule.
+
+{{< tabs >}}
+{{% tab "Add scanning rule from the library rules" %}}
+
+The Scanning Rule Library contains predefined rules for detecting common patterns such as email addresses, credit card numbers, API keys, authorization tokens, and more.
+
+1. In the **Add library rules to the scanning group** section, select the library rules you want to use.
+1. In the **Define rule target and action** section, select if you want to scan the **Entire Event** or **Specific Attributes**.
+    - If you are scanning the entire event, you can optionally exclude specific attributes from getting scanned.
+    - If you are scanning specific attributes, specify which attributes you want to scan.
+{{% sds-scanning-rule %}}
+1. Click **Add Rules**.
+
+{{% /tab %}}
+{{% tab "Add a custom scanning rule" %}}
+
+You can create custom scanning rules using regex patterns to scan for sensitive data.
+
+1. In the **Define match conditions** section, specify the regex pattern to use for matching against events in the **Define regex** field. Enter sample data in the **Regex tester** field to verify that your regex pattern is valid.   
+    Sensitive Data Scanner supports Perl Compatible RegEx (PCRE), but the following patterns are not supported:
+    - Backreferences and capturing sub-expressions (lookarounds)
+    - Arbitrary zero-width assertions
+    - Subroutine references and recursive patterns
+    - Conditional patterns
+    - Backtracking control verbs
+    - The \C "single-byte" directive (which breaks UTF-8 sequences)
+    - The \R newline match
+    - The \K start of match reset directive
+    - Callouts and embedded code
+    - Atomic grouping and possessive quantifiers
+1. For **Create keyword dictionary**, add keywords to refine detection accuracy when matching regex conditions. For example, if you are scanning for a sixteen-digit Visa credit card number, you can add keywords like `visa`, `credit`, and `card`. You can also require that these keywords must be within a specified number of characters of a match. By default, keywords must be within 30 characters before a matched value.
```

Comment:
You can create keyword dictionary for library rules too, so I would add this to the library rules section as well!

Reply:
Added, thanks!